index.html

```diff
@@ -6396,7 +6401,9 @@ <h2 id="AudioWorklet-Examples">
           <dd>
             <p>
               A reference distance for reducing volume as source moves further
-              from the listener. The default value is 1.
+              from the listener. The default value is 1. A
+              <code>TypeError</code> exception must be thrown if this is set to
+              a non-positive value.
```

Comment:
Usually "IndexSizeError" for that kind of things, although it's weird. RangeError does not exist any more apparently.